docs/src/main/asciidoc/_elytron/Credential_Store.adoc

```diff
@@ -235,6 +235,80 @@ ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBMfncZuH
+                                                   b3BlbnNzaC1rZXktdjEAAAAACmFlczI1Ni1jdHIAAAAGYmNyeXB0AAAAGAAAABCdRswttV
+                                                   UNQ6nKb6ojozTGAAAAEAAAAAEAAABoAAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlz
+                                                   dHAyNTYAAABBBAKxnsRT7n6qJLKoD3mFfAvcH5ZFUyTzJVW8t60pNgNaXO4q5S4qL9yCCZ
+                                                   cKyg6QtVgRuVxkUSseuR3fiubyTnkAAADQq3vrkvuSfm4n345STr/i/29FZEFUd0qD++B2
+                                                   ZoWGPKU/xzvxH7S2GxREb5oXcIYO889jY6mdZT8LZm6ZZig3rqoEAqdPyllHmEadb7hY+y
+                                                   jwcQ4Wr1ekGgVwNHCNu2in3cYXxbrYGMHc33WmdNrbGRDUzK+EEUM2cwUiM7Pkrw5s88Ff
+                                                   IWI0V+567Ob9LxxIUO/QvSbKMJGbMM4jZ1V9V2Ti/GziGJ107CBudZr/7wNwxIK86BBAEg
+                                                   hfnrhYBIaOLrtP8R+96i8iu4iZAvcIbQ==
+                                                   -----END OPENSSH PRIVATE KEY-----", passphrase=secret)
+{"outcome" => "success"}
+----
+
+NOTE: The parameter passphrase is optional and can be empty if no passphrase was used to encrypt the key.
+
+or from a file containing a private key in OpenSSH format:
+
+[source,options="nowrap"]
+----
+[standalone@localhost:9990 /] /subsystem=elytron/credential-store=mycredstore:import-key-pair(alias=example, openssh-private-key-location=/home/user/.ssh/id_rsa, passphrase=secret)
+{"outcome" => "success"}
+----
+
+If the key is specified in PKCS format rather than OpenSSH format, both the private and public key must be defined and the PKCS private key must not be encrypted with a passphrase. Key pair credentials in PKCS format can be as well imported by providing the private key and public key value:
+
+[source,options="nowrap"]
+----
+[standalone@localhost:9990 /] /subsystem=elytron/credential-store=mycredstore:import-key-pair(alias=example, private-key="-----BEGIN PRIVATE KEY-----
+                                                   MIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQgj+ToYNaHz/pISg/Z
+                                                   I9BjdhcTre/SJpIxASY19XtOV1ehRANCAASngcxUTBf2atGC5lQWCupsQGRNwwnK
+                                                   6Ww9Xt37SmaHv0bX5n1KnsAal0ykJVKZsD0Z09jVF95jL6udwaKpWQwb
+                                                   -----END PRIVATE KEY-----",
+                                                   public-key="-----BEGIN PUBLIC KEY-----
+                                                   MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEp4HMVEwX9mrRguZUFgrqbEBkTcMJ
+                                                   yulsPV7d+0pmh79G1+Z9Sp7AGpdMpCVSmbA9GdPY1RfeYy+rncGiqVkMGw==
+                                                   -----END PUBLIC KEY-----")
+{"outcome" => "success"}
+----
+
+or from files containing a private and public key:
```

Comment:
Would be good to add something to the beginning of the line here too, e.g., Key pair credentials in PCKS format can also be imported from files...